src/movies.js

```diff
@@ -2,14 +2,137 @@
 
 // Iteration 1: Ordering by year - Order by year, ascending (in growing order)
 
+function orderByYear(array) {
+  let arrYear = array.slice();
```

Comment:
Although slice works good in this case, semantically, maybe it would make more sense to use array spreading to copy the array ( let arrYear = [...array] ), because you are not trying to slice a particular part of the array.